phpcs.xml

```diff
@@ -3,5 +3,8 @@
     <rule ref="vendor/keboola/coding-standard/src/ruleset.xml">
         <!-- Don't remove @internal doc comments from ValueObjects __construct -->
         <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.UselessDocComment"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingTraversableTypeHintSpecification"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingTraversableTypeHintSpecification"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingTraversableTypeHintSpecification"/>
```

Comment:
tohle zatím budeme ignorovat... chce to určit jakýho typu jsou pole např. `string[]`

Reply:
ok